### ISSUE TITLE
Retire MolSim

### ISSUE THREAD

**Issue by cortner** [JuliaMolSim/MolSim#6]
It appears that - aside from DFTK.jl - I am the only one using this registry. DFTK.jl is more than ready to move to General (probably has been for a while, may it's already in General?).

In the meantime I occasionally get grumpy users asking what the point of the local registry is. 

I am therefore wondering out loud ... should we just 
(1) shut down the MolSim registry
(2) move general-use and mature packages to General 
(3) For my own group's purposes / the ACE eco-system I create a new local registry

thoughts? @mfherbst ? anybody else?

**Comment by mfherbst** [comment]
> (probably has been for a while, may it's already in General?).

Yes, most people have been using it exclusively from General.

> In the meantime I occasionally get grumpy users asking what the point of the local registry is.

Yes, opinions differ here. I honestly have not found custom registries useful in my everyday Julia coding, so I have not really been using them a lot. But I see why they could be useful, so I don't really have a strong opinion.

Regarding `MolSim`, I'm not really depending on it, but it also does not bother me, so either way (closing or not) is fine with me. :)

**Comment by antoine-levitt** [comment]
I thought you were the one pushing for custom registries @cortner , so if you want to close it it's fine by me. Indeed DFTK uses General.

**Comment by cortner** [comment]
Yes, that was my assumption as well (and now with massive data to support it) ...  but given I'm not the sole owner of this org I didn't feel I can take this step on my own. 

So unless I hear any other optnions I will now take the following steps: 
- Move all my packages to a group-internal registry
- State on the README that this registry is no longer in use but will remain available for x amount of time. 
- After that time, archive the registry repo.
